source/includes/api/_geostore.md

```diff
@@ -125,395 +145,845 @@ curl -X POST https://api.resourcewatch.org/v1/geostore \
+                                        "type": "MultiPolygon"
+                                    },
+                                    "type": "Feature"
+                                }
+                            ]
+                        },
+                        "hash": "8f77fe62cf15d5098ba0ee11c5126aa6",
+                        "provider": {},
+                        "areaHa": 54935531.53854849,
+                        "bbox": [
+                            -5.14375114440907,
+                            41.3337516784668,
+                            9.5604162216186,
+                            51.08939743042
+                        ],
+                        "lock": false,
+                        "info": {
+                            "use": {},
+                            "iso": "FRA",
+                            "name": "France"
+                        }
+                    }
+                }
+            }
+        }
+    ],
+    "info": {
+        "found": 2,
+        "foundIds": [
+            "35a6d982388ee5c4e141c2bceac3fb72",
+            "8f77fe62cf15d5098ba0ee11c5126aa6"
+        ],
+        "returned": 2
+    }
+}
+```
+
+#### Errors for getting multiple geostore objects by id
+
+Error code     | Error message       | Description
+-------------- | ------------------- | -----------------------------------------------
+404            | Geostore not found. | A geostore with the id provided was not found.
 
-Is possible obtain the geostore of a World Database on Protected Areas of the world. You only need the id of the protected area (WDPA).
-[World Database on Protected Areas web](https://www.protectedplanet.net/)
 
-`GET https://api.resourcewatch.org/v1/geostore/wdpa/<wdpaId>`
+### Get all Geostore ids, names and country codes
 
+All of the Geostore ids, names, and country ISO 3166-1 alpha-3 code values available from the GADM data source in the Geostore can be retrieved via the GET `geostore/admin/list` endpoint, which returns an array of objects with the properties `geostoreId`, `name`, and `iso`.
+
+> Example URL request
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/geostore/wdpa/<wdpaId>
+curl -X GET https://api.resourcewatch.org/v2/geostore/admin/list
 ```
 
-> Real example obtaining the geostore of 'Sierra de Guadarrama' protected area
+> Example response
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/geostore/wdpa/555538160
+```json
+{
+    "data": [{
+        "geostoreId": "35a6d982388ee5c4e141c2bceac3fb72",
+        "iso": "ESP",
+        "name": "Spain"
+    }, {
+        "geostoreId": "8f77fe62cf15d5098ba0ee11c5126aa6",
+        "iso": "FRA",
+        "name": "France"
+    }, {
+        "geostoreId": "1d568c183033da6c17cc28c4aecf1bcf",
+        "iso": "cod",
+        "name": "Democratic Republic of the Congo"
+    }]
+}
 ```
 
-### From land use areas
+*Note - This endpoint does not return a unique array of iso codes and names.*
+
+### Get geostore by country code
```

Comment:
I would put all of these (country code, ADM1, ADM2, WDPA, land use type) in their own section called something like "Getting predefined geostores for commonly used geometries". Point out that this is essentially a repository of useful geometries that is provided as a utility function of sorts. The fact that they are stored with the same underlying mechanism as custom geostores is really just a technical matter.